opentelemetry-exporter-geneva/geneva-uploader/src/config_service/client.rs

```diff
@@ -498,21 +498,24 @@ fn extract_endpoint_from_token(token: &str) -> Result<String> {
 
     // Base64-decode the JWT payload (2nd segment of the token).
     // Some JWTs may omit padding ('='), so we restore it manually to ensure valid Base64.
-    // This is necessary because the decoder (URL_SAFE_NO_PAD) expects properly padded input.
+    // This is necessary because the decoder (URL_SAFE) expects properly padded input.
```

Comment:
What are these changes? Are these needed for correctness?

Reply:
Was getting error in decoding token on MacOS with `URL_SAFE_NO_PAD` + locally generated token. `URL_SAFE` worked both in MacOS and WSL, for remote and locally generated token. But have reverted this change for now, will raise separate PR for this, if necessary.